setup.py

```diff
@@ -57,6 +57,18 @@
     # https://packaging.python.org/specifications/core-metadata/#description-optional
     long_description=long_description,  # Optional
 
+    # Denotes that our long_description is in Markdown; valid values are
+    # text/plain, text/x-rst, and text/markdown
+    #
+    # Optional if long_description is written in rst but required for
```

Comment:
Could you change `rst` here to `reStructuredText`? Since newcomers might not know what "rst" stands for.